packages/bcer-retailer-app/app/src/components/form/validations/vBusinessLocation.ts

```diff
@@ -48,7 +49,19 @@ export const Initial = {
 };
 
 export const Validation = yup.object({
-  addressLine1: yup.string().test('length', 'The address must be less than 100 characters.', val => val?.length <= 100).required('The address of your place of business is required'),
+  addressLine1: yup.string().required('The address of your place of business is required')
+    .test('exists', `Address couldn't be found or is incorrect`,async (val) => {
+      try {
+        const {data} = await Axios.get(`https://geocoder.api.gov.bc.ca/addresses.json?minScore=70&maxResults=1&echo=false&autoComplete=false&brief=false&matchPrecision=occupant,unit,site,civic_number,block&addressString=${val}`)

```

Comment:
I see that we've got this address a few places in the codebase now. I think it probably deserves being a constant somewhere :) 

Also think it could be good to move this lookup into a utility function so it can be reused whenever needed

Reply:
@roy-sagar11 @simensma-fresh one thing to look out for - we could make a constant for the https://geocoder....addresses.json part, but the rest of the query differs between implementations. we would need to concatenate the string request before use